src/components/grid-list/grid-list.ts

```diff
@@ -18,7 +19,7 @@ import {
     FixedTileStyler
 } from './tile-styler';
 import {MdGridListColsError} from './grid-list-errors';
-import {Dir} from '@angular2-material/core/rtl/dir';
```

Comment:
Oops, was trying to make the Webstorm gods happy (red underlines).  I'll revert.